operators/endpointmetrics/controllers/observabilityendpoint/ocp_monitoring_config.go

```diff
@@ -394,16 +394,20 @@ func createOrUpdateClusterMonitoringConfig(
 		// check if alertmanagerConfigs exists
 		if foundClusterMonitoringConfiguration.PrometheusK8sConfig.AlertmanagerConfigs != nil {
 			additionalAlertmanagerConfigExists := false
-			for _, v := range foundClusterMonitoringConfiguration.PrometheusK8sConfig.AlertmanagerConfigs {
+			var atIndex int
+			for i, v := range foundClusterMonitoringConfiguration.PrometheusK8sConfig.AlertmanagerConfigs {
```

Comment:
Would it be easier here to have a func that just checks for the alertmanager URL that we care about, and returns true if it finds the object. Or conversely if we always just add it when there's a reconcile. Is there a cost to updating this field? (Does alertmanager restart)